bumble/profiles/vcp.py

```diff
@@ -88,6 +88,7 @@ def __init__(
         muted: int = 0,
         change_counter: int = 0,
         volume_flags: int = 0,
+        included_services: List[gatt.Service] = [],
```

Comment:
Should not be list with a default value because it's mutable. (We forgot to change this for `gatt.Service` as well.

```suggestion
        included_services: Sequence[gatt.Service] = (),
```